src/amo/components/AuthenticateButton/index.js

```diff
@@ -99,6 +100,7 @@ export class AuthenticateButtonBase extends React.Component<InternalProps> {
     // https://github.com/mozilla/addons-frontend/issues/1904
     return (
       <Button
```

Comment:
Both will not work at same time i guess ;)

Reply:
Yes, got it. Thank you so much @championshuttler